terraform/aptos-node/aws/kubernetes.tf

```diff
@@ -41,6 +41,18 @@ resource "kubernetes_storage_class" "io1" {
   }
 }
 
+resource "kubernetes_storage_class" "io2" {
+  metadata {
+    name = "io2"
+  }
+  storage_provisioner = "kubernetes.io/aws-ebs"
+  volume_binding_mode = "WaitForFirstConsumer"
+  parameters = {
+    type      = "io2"
+    iopsPerGB = "50"
```

Comment:
Need to double check if "io2" provides iopsPerGB parameter or it should be configured in a different way.